Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation.yml

```
@@ -764,14 +764,139 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "14":
+    continueonerrortype: ""
+    id: "14"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    loop:
+      exitCondition: ""
+      iscommand: false
+      max: 100
+      wait: 1
+    nexttasks:
+      '#none#':
+      - "4"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      ASM Rule ID:
+        simple: ${alert.asmattacksurfaceruleid}
+      Account ID:
+        simple: ${alert.asmcloud.project}
+      Assume Role:
+        simple: ${inputs.AWSAssumeRoleName}
+      Instance ID:
+        complex:
+          accessor: id
+          filters:
+          - - left:
+                iscontext: true
+                value:
+                  simple: alert.asmsystemids.type
+              operator: isEqualString
+              right:
+                value:
+                  simple: ASSET-ID
+          root: alert.asmsystemids
+      Region:
+        complex:
+          accessor: region
+          root: alert.asmcloud
+          transformers:
+          - operator: FirstArrayElement
+    separatecontext: true
+    skipunavailable: false
```

```suggestion
    skipunavailable: true
```
This should fix the validation error and be the last change I will request

---

Thank you, this worked!!